lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -52,6 +61,26 @@ static inline void skip_command_prefix(const char **cmd)
 	(*cmd)++;
 }
 
+static inline bool check_response_for_forced_string(const char *tmpstr)
+{
+	bool retval = false;
+
+	if (!strncmp(tmpstr, "%HWVERSION", AT_CMD_HWVERSION_LEN)) {
+			retval = true;
```

Comment:
The indentation is too deep, should be one tab less.

Reply:
Fixed.